.github/workflows/codeql-analysis.yml

```
@@ -67,13 +67,13 @@ jobs:
       if: ${{ matrix.language == 'java' && github.repository_owner == 'gradle' }}
       run: ./gradlew --init-script .github/workflows/codeql-analysis.init.gradle -DcacheNode=us -S testClasses -Dhttp.keepAlive=false
       env:
-        # Set the DEVELOCITY_ACCESS_KEY so that Gradle Build Scans are generated
+        # Set the DEVELOCITY_ACCESS_KEY so that a Gradle Build Scan is generated
         DEVELOCITY_ACCESS_KEY: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
         # Potential stop-gap solution for ReadTimeout issues with the Gradle Build Cache
         # https://gradle.slack.com/archives/CHDLT99C6/p1636477584059200
         GRADLE_OPTS: -Dhttp.keepAlive=false
 
-    - name: Compile with Gradle without Build Scan
+    - name: Compile with Gradle without a Build Scan
```

```suggestion
    - name: Compile with Gradle without publishing a Build Scan
```

I would use a word in front of the `a Build Scan`.